### ISSUE TITLE
Local Npm module "js2coffee" not found

### ISSUE THREAD

**Issue by FarhadG** [jonschlinkert/grunt-js2coffee#3]
I'm not sure what's breaking this, however, I'm constantly getting this error with these specs:

npm: 1.3.14
grunt: 0.4.2
js2coffee: 0.2.3

Js2coffee has been installed locally and the package.json file all seems proper, so I'm curious if you've had this error before.

Thanks for the help, in advance.

**Comment by jonschlinkert** [comment]
No, I haven't had the error before. I'll look into it. thanks!

**Comment by FarhadG** [comment]
Awesome! Thanks for the quick response. Yeah, I can't seem to track down what's causing this error, at all.

**Comment by ff6347** [comment]
I'm getting the same issue with the same setup.

**Comment by jonschlinkert** [comment]
Okay, thanks for the feedback. I'm planning on fixing this sometime this weekend

**Comment by jonschlinkert** [comment]
I spent some time on this today, and I can't reproduce the error. I tried pulling down the project from npm and git clone, then installed dependencies from scratch and it worked fine for me. Could this be a js2coffee issue?

**Comment by jonschlinkert** [comment]
If you guys have any ideas on this I'd be happy to take another look

**Comment by FarhadG** [comment]
That's very strange, because js2coffee seems to work just fine. Any suggestions to look into?

**Comment by jonschlinkert** [comment]
No clue, sorry. I tried for about an hour. I even tried clearing my cache, no errors

**Comment by FarhadG** [comment]
Are you using nvm? Perhaps there may be something there

**Comment by ff6347** [comment]
js2coffee is not working for me. When I execute this from the root of my project I get an empty file test.coffee.

```
node_modules/js2coffee/out/bin/js2coffee test.jsx > test.coffee
```

Or am I executing it wrong?

**Comment by jonschlinkert** [comment]
if that's the case, then I'm not sure how this could be related to this Grunt plugin. You're directly using the cli for js2coffee in your example (and I'm assuming the `test.jsx` is a typo).

**Comment by ff6347** [comment]
No. test.jsx is not a typo. I'm using Adobe extend script. I also tried to use the grunt plugin and I got the error "js2coffee not found".

**Comment by jonschlinkert** [comment]
Ah, go it. 

> I also tried to use the grunt plugin and I got the error "js2coffee not found".

But the fact that this doesn't work: `node_modules/js2coffee/out/bin/js2coffee test.jsx > test.coffee` is confusing. mabye try pulling down js2coffee to a different dir?

**Comment by ff6347** [comment]
I can try later.

**Comment by ff6347** [comment]
Hi. I pulled [js2coffee](http://js2coffee.org/#info) to a different dir.  
One thing is it only handles .js files and not .jsx. But then it works.
